simcon_project/templates/researcher_view.html

```diff
@@ -1,35 +1,33 @@
+{% extends "researcher_base.html" %}
+{% load render_table from django_tables2 %}
+{% load static %}
+{% block nav-researcherView %}{% endblock %} 
+{% block content %}
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="Created for Simulated Conversations Senior Capstone Project">
-
-    <!-- prevent line breaks when using form fields -->
-    <style type="text/css">
-    form {display:inline; margin:0px; padding:0px;}
-    </style>
-    
-
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
```

Comment:
Since you are extending researcher_base.html where these imports are already done, do you need them here again?

Reply:
they render separately so we need both

Reply:
Oof that's annoying. Ok